src/Business/Seeds/DetectionSeed.cs

```diff
@@ -0,0 +1,34 @@
+namespace Business.Seeds
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Business.Detection.Common.Repositories;
+    using Business.Detection.Creating.Models;
+
+    public class DetectionSeed : ISeed
+    {
+        private readonly IDetectionRepository _repository;
+        private readonly IDataFactory _dataFactory;
+
+        public DetectionSeed(IDetectionRepository repository, IDataFactory dataFactory)
+        {
+            _repository = repository;
+            _dataFactory = dataFactory;
+        }
+
+        public void Execute()
+        {
+            var missingDetections = MissingDetections();
+            foreach (var missingDetection in missingDetections)
+            {
+                _repository.Insert(missingDetection);
+            }
+        }
+
+        private IEnumerable<MCreateDetection> MissingDetections()
```

Comment:
Should we rename the `MCreateDetection` model into `MCreateDetections`? Our Insert from repo will insert a list of detections not a single one and that`s why we need to rename the model.

Reply:
done